### ISSUE TITLE
Perinasal hyperpigmentation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11679]
# Perinasal hyperpigmentation

## Proposed definition

Increased pigmentation of the skin in the region surrounding the nose.

## Parent term

- `HP:0000953 Hyperpigmentation of the skin`

## Synonyms

- Hyperpigmentation around the nose
- Perinasal skin darkening

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617
@evanc1648

**Comment by pnrobinson** [comment]
Periumbilical hyperpigmentation

## Proposed definition

Increased pigmentation of the skin in the region surrounding the umbilicus.

## Parent term

- `HP:0000953 Hyperpigmentation of the skin`

## Synonyms

- Umbilical hyperpigmentation

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617

**Comment by pnrobinson** [comment]
# Periungual hyperpigmentation

## Proposed definition

Increased pigmentation of the skin immediately surrounding the fingernails or toenails.

## Parent term

- `HP:0000953 Hyperpigmentation of the skin; HP:0100803 Abnormal periungual morphology`

## Synonyms

- Hyperpigmentation around the nails
- Periungual darkening

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617

**Comment by pnrobinson** [comment]
@evanc1648

Adding new term: 
- ID: HP:6001519
- label: Perinasal hyperpigmentation
- parents: [Hyperpigmentation of the skin (HP:0000953)](https://hpo.jax.org/app/browse/term/HP:0000953)
- pmid: PMID:20301443
- def: Increased pigmentation of the skin in the region surrounding the nose.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11679
- synonyms: Perinasal skin darkening(NONE)|Hyperpigmentation around the nose(NONE)

**Comment by pnrobinson** [comment]
@evan1648 
Adding new term: 
- ID: HP:6001520
- label: Periumbilical hyperpigmentation
- parents: [Hyperpigmentation of the skin (HP:0000953)](https://hpo.jax.org/app/browse/term/HP:0000953)
- pmid: PMID:24878829
- def: Increased pigmentation of the skin in the region surrounding the umbilicus (belly button).
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11679
- synonyms: Umbilical hyperpigmentation(NONE)

**Comment by pnrobinson** [comment]
@evan1648
Adding new term: 
- ID: HP:6001521
- label: Periungual hyperpigmentation
- parents: [Hyperpigmentation of the skin (HP:0000953)](https://hpo.jax.org/app/browse/term/HP:0000953)
- pmid: PMID:3410999
- def: Increased pigmentation of the skin immediately surrounding the fingernails or toenails.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11679
- synonyms: Periungual darkening(NONE)|Hyperpigmentation around the nails(NONE)
